kernel/packages/shared/catalogs/sagas.ts

```diff
@@ -306,12 +294,14 @@ function* fetchWearablesV1(filters: WearablesRequestFilters) {
   const platformCatalog = yield select(getPlatformCatalog)
   const exclusiveCatalog = yield select(getExclusiveCatalog)
 
-  let response: Wearable[]
+  let response: PartialWearableV2[]
   if (filters.wearableIds) {
     // Filtering by ids
     response = filters.wearableIds
       .map((wearableId) =>
-        wearableId === 'dcl://base-avatars/SchoolShoes' ? 'dcl://base-avatars/Moccasin' : wearableId
+        wearableId === 'urn:decentraland:off-chain:base-avatars:SchoolShoes'
+          ? 'urn:decentraland:off-chain:base-avatars:Moccasin'
+          : wearableId
```

Comment:
What's this hack for? It's still relevant?

Reply:
Yes, this is still necessary. The wearable-api has both `dcl://base-avatars/Moccasin` and `dcl://base-avatars/moccasin`. So we converted `Moccasin` to `SchoolShoes` during the migration. But if we need to use the v1 catalog, we need to translate it back to `Moccasin` for it work